src/app.js

```diff
@@ -0,0 +1,15 @@
+const path = require("path");
+const express = require("express");
+const app = express();
+const router = require("./router");
+app.use(express.static(path.join(__dirname, "..", "client", "build")));
+app.use(express.json());
+app.use(express.urlencoded());
+app.use(router);
+
+app.get('*',(req,res)=>{
+
```

Comment:
Replace `app.get("*")` with `app.get("/")` Since with the start the user will get the landing page no matter what they  typed